app/misc/repo.ts

```diff
@@ -261,7 +261,9 @@ function openRepository() {
   function refreshAll(repository) {
     document.getElementById('spinner').style.display = 'block';
     let branch;
-    bname = [];
+    bname = {};
```

Comment:
I dont think this affected the code at all, but bname should have been a object, not an array.